docker/runtime/bash/http-trigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM bitnami/minideb:jessie 
+
+RUN apt-get update
+
+ADD install_dependencies.sh /bin/
+ADD requirements /tmp
+RUN /bin/install_dependencies.sh
```

Comment:
I'm confused by this (and the corresponding exception added to `GetFunctionImage()`). Doesn't `install_dependencies.sh` _need_ to run in the init container at Function-creation-time, not here at runtime-image-build-time?

Reply:
it actually need to be run in the first boot of the runtime container. It cannot be run at build time since the requirements file won't be there yet nor using an init container since we cannot know where the output will be stored (so we could mount it in the runtime container). Note that the `requirements` file will be mounted so we don't need to add it. This should look something like:

```
FROM bitnami/minideb:jessie 

RUN apt-get update

RUN install_packages python-pip
RUN pip install bottle==0.12.13 cherrypy==8.9.1 wsgi-request-logger prometheus_client

ADD install_dependencies.sh /bin/
ADD kubeless-shell.py /
EXPOSE 8080
ENTRYPOINT ["/bin/install_dependencies.sh"]
CMD ["python", "/kubeless-shell.py"]
```